arm-software/embedded/test/multilib/armv6m.test

```diff
@@ -1,13 +1,13 @@
 # RUN: %clang -print-multi-directory --target=armv6m-none-eabi -mfpu=none | FileCheck %s --check-prefix=CHECK-NOFP-EXN-RTTI-UNALIGNED
-# CHECK-NOFP-EXN-RTTI-UNALIGNED: arm-none-eabi/armv6m_soft_nofp_exn_rtti_unaligned{{$}}
+# CHECK-NOFP-EXN-RTTI-UNALIGNED: arm-none-eabi/armv6m_soft_nofp_exn_rtti{{$}}
 # CHECK-NOFP-EXN-RTTI-UNALIGNED-EMPTY:
 
 # RUN: %clang -print-multi-directory --target=armv6m-none-eabi -mfpu=none -mno-unaligned-access | FileCheck %s --check-prefix=CHECK-NOFP-EXN-RTTI
 # CHECK-NOFP-EXN-RTTI: arm-none-eabi/armv6m_soft_nofp_exn_rtti{{$}}
 # CHECK-NOFP-EXN-RTTI-EMPTY:
 
 # RUN: %clang -print-multi-directory --target=armv6m-none-eabi -mfpu=none -fno-exceptions -fno-rtti | FileCheck %s --check-prefix=CHECK-NOFP-UNALIGNED
```

Comment:
Change CHECK-NO-FP-UNALIGNED to CHECK-NO-FP. That is, remove the "-UNALIGNED" suffix.